Topics/Software_Tools/Markdown/Markdown.md

```diff
@@ -0,0 +1,68 @@
+<img width="96" alt="bold4" src="https://github.com/learning-software-engineering/learning-software-engineering.github.io/assets/74106672/1a42367b-8ffb-4caa-8467-5eab33d10991">
+
+#### Italics 
+Italicizing text is done in similar fashion to bolding text. To italicize words or phases, surround the text with a single asterick (*) or underscore (_) on each side. To italicize certain characters within a word, surround the characters with a single asterick on each side. An example is shown below.  
+<img width="182" alt="italicize1" src="https://github.com/learning-software-engineering/learning-software-engineering.github.io/assets/74106672/b4c369c8-cee5-4835-935c-d96db4f71d71">
+<img width="162" alt="italicize2" src="https://github.com/learning-software-engineering/learning-software-engineering.github.io/assets/74106672/00e0e340-c5a3-4be9-9de9-0b69113583c6">
+
+
+#### Bold and Italics
+To bolden and italicize text at the same time, surround the text with a pair of any of the following options displayed below, where the first two options are three astericks and three underscores. To bolden and italicize characters within a word, surround the characters with three astericks. An example is shown below.   
+<img width="183" alt="both" src="https://github.com/learning-software-engineering/learning-software-engineering.github.io/assets/74106672/dd07d4a2-2cc7-4473-a913-3b8e0a9e4f6b">
+<img width="142" alt="both2" src="https://github.com/learning-software-engineering/learning-software-engineering.github.io/assets/74106672/2f6c3381-9cd6-48dd-8dc0-134636decf38">
+
+
+### Lists
+To create ordered lists, number each item in the list with a number followed by a period. An example is shown below.   
+<img width="87" alt="unordered1" src="https://github.com/learning-software-engineering/learning-software-engineering.github.io/assets/74106672/52b6b667-f1f4-4e93-a491-084c90b7bda0">
+<img width="94" alt="unordered2" src="https://github.com/learning-software-engineering/learning-software-engineering.github.io/assets/74106672/2df3b800-4f76-48ea-9307-dcc65b0202d3">
+
+Unordered lists can be done by constitently using either dashes, astericks, or plus signs before each item. Indented items can be done by hitting tab before using the dash, asterick, or plus sign.  
+
+### Paragraphs
+To create paragraphs, simply use at least one blank line in between lines of text. An example is shown below.   
+<img width="115" alt="paragraph1" src="https://github.com/learning-software-engineering/learning-software-engineering.github.io/assets/74106672/bcc50f47-7691-4b41-b947-52cb277d7f05">
+<img width="118" alt="paragraph2" src="https://github.com/learning-software-engineering/learning-software-engineering.github.io/assets/74106672/9dbc7b5a-7121-4a6f-a413-d7cd6b1e642f">
+
+There are many more features used in Markdown, these are only a few. To read more about Markdown in regards to syntax, or other inquiries, consult this link: https://www.markdownguide.org/
```

Comment:
Maybe mention some of the other features?

Reply:
I agree! I named a few that I thought were useful and/or fun.